app/src/main/res/layout/fragment_form_customer_contact.xml

```diff
@@ -75,7 +75,8 @@
                         <EditText
                             android:hint="@string/optional_mobile"
                             android:id="@+id/et_mobile"
-                            android:inputType="text"
+                            android:inputType="number"
+                            android:maxLength="10"
```

Comment:
@PatelVatsalB21 If I'm not wrong then there is no international limit of phone number length. Actually phone number length vary from region to region. So limiting the length to 10 wouldn't be good idea.

Reply:
Oh sorry it should be 15. Shouldn't it?

Reply:
Actually there is no correct solution because in some apps I saw a feature where the max length varies according to region. So setting a static maxlength wouldn't solve this issue exactly.

Reply:
I found max length by international telecommunication union is 15 should i make it 15?

Reply:
As I told you above making it 15 will solve the problem for the region with phone number length 15, what about regions with phone number length less than 15. @jawidMuhammadi what are views on this. I think make it 15 for now, maybe later we can add a feature for this.

Reply:
I thing there's a better workaround for this. We can make a map of country codes to the countries. We get the country from the user earlier while editing or creating a new customer, from there we can get the country code which can be appended to the 10 digit mobile number. This way , we can solve the problem of varied length of mobile numbers.

Reply:
@EGOR-IND @PatelVatsalB21 @jawidMuhammadi please correct me if I am wrong anywhere.

> I thing there's a better workaround for this. We can make a map of country codes to the countries. We get the country from the user earlier while editing or creating a new customer, from there we can get the country code which can be appended to the 10 digit mobile number. This way , we can solve the problem of varied length of mobile numbers.

Reply:
> I thing there's a better workaround for this. We can make a map of country codes to the countries. We get the country from the user earlier while editing or creating a new customer, from there we can get the country code which can be appended to the 10 digit mobile number. This way , we can solve the problem of varied length of mobile numbers.

Yup that's a good solution, I think so.

Reply:
Let it be simple or else like countries list in details it won't work offline. I will make it Max 15 digits as allowed. Above 15 no country is allowed to use numbers.

Reply:
I think you misunderstood me. What I am saying is that we will add a map which would be static in the code base , so while using the country code from that map doesn't need the user to be online. I hope that makes sense.